.github/workflows/tests.yml

```diff
@@ -21,44 +21,60 @@ jobs:
         run: .github/workflows/xcode_select.sh
       - name: Build and Test
         run: |
-          # Host config
-          bazelisk test --local_test_jobs=1 -- //... -//tests/ios/...
+          # non-iOS tests
+          bazelisk test \
+            --config=ci \
+            -- \
+            //... \
+            -//tests/ios/...
```

Comment:
nit: I like this to be on a single line, so that it's easier to copy it into to terminal. Same as other commands as well.